lldb-eval/parser_context.h

```diff
@@ -61,7 +61,7 @@ class ParserContext {
   virtual clang::SourceManager& GetSourceManager() const = 0;
   virtual TypeSP ResolveTypeByName(const std::string& name) const = 0;
   virtual std::unique_ptr<IdentifierInfo> LookupIdentifier(
-      const std::string& name) const = 0;
+      const std::string& name) = 0;
```

Comment:
Why did it lose `const`?

Reply:
It uses `GetEmptyType()` indirectly, which isn't `const`. I wasn't sure if that was intended or not.
Marked all methods `const`.